src/cockpit/channel.py

```diff
@@ -469,6 +483,7 @@ async def read(self) -> 'bytes | None':
             if isinstance(item, Mapping):
                 self.send_pong(item)
             else:
+                self.send_ack(item)
```

Comment:
Should also update the "three possibilities" documentation with this?

Reply:
This is part of handling the possibility that we received data.  I can mention that we send an ack in that case, though.